app/resources/css-default.template

```diff
@@ -1133,3 +1139,18 @@ table.api-keys td.empty {
     padding: 5px;
   }
 }
+
+.card {
+  background: #ffffff;
+  border: 1px solid #dadada;
+  box-shadow: 2px 2px 3px 0px #AAA;
+  padding: 10px;
+  margin: 10px;
+  border-radius: 3px;
+}
+
+.card h1 {
+  margin: auto;
+  padding: 10px;
+  text-align: center;
```

Comment:
Optional: Looks like we don't centerize captions in Material Design:
https://material.google.com/components/cards.html#cards-actions
So it would be better to stop centerizing here.